packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -118,7 +118,7 @@ class NavigationRail extends StatefulWidget {
     this.useIndicator,
     this.indicatorColor,
     this.indicatorShape,
-  }) :  assert(destinations.length >= 2),
+  }) :  assert(destinations.length >= 0),
```

Comment:
can remove this assert

Reply:
[4021daf](https://github.com/flutter/flutter/pull/152972/commits/4021dafa6295a8da80decaedc6d1da5548e2b3d7)